src/utils.js

```diff
@@ -74,24 +75,25 @@ export const getICOs = () => Object.keys(config.ICOs).map((icoKey) => {
 
 export const getValueOrNotAvailable = (props, input) => props && props[input] ? props[input] : 'Not Available';
 
+export const getValueOrDefault = value => value || 'Not Available';
+
 export const trimString = value => value.replace(/ /g, '');
 
 export const getICOLogs = (blockRange, icoConfig, icoContract, callback) => {
   console.log(`Start scanning for block range ${blockRange}`, icoContract.address);
-  /* if (typeof localStorage !== 'undefined' && localStorage.getItem(address)) {
-    console.log(`${address} cached already.`);
-    return callback(null, JSON.parse(localStorage.getItem(address)));
-  }*/
   const address = icoContract.address;
   const eventName = blockRange[2];
   const event = icoConfig.events[eventName];
+
   const filter = icoContract[eventName](event.customArgs || {}, {
     fromBlock: blockRange[0],
     toBlock: blockRange[1],
   });
   filter.stopWatching(() => {});
 
-  jQuery.ajax({
+  const $ = process.env.NODE_ENV === 'test' ? testJQuery.jQuery : jQuery;
```

Comment:
what about writing our own module called `jquery`, always using it for accesing jquery and putting that check there? This will avoid having multilple checks like this in the code base?